site/download.md

```diff
@@ -39,7 +39,7 @@ docker run -it --rm --name rabbitmq -p 5672:5672 -p 15672:15672 rabbitmq:3-manag
 
 ## Kubernetes
 
- * VMware [Tanzu RabbitMQ on Kubernetes](https://pivotal.io/pivotal-rabbitmq-on-kubernetes) (currently in beta)
+ * [RabbitMQ Cluster Operator](/kubernetes-operator.html) ([Source Code](https://github.com/rabbitmq/cluster-operator))
```

Comment:
Please make "Source Code" say "GitHub repository". This is not shareware but actual open source software ;)